kratos/python_scripts/restart_utility.py

```diff
@@ -39,11 +40,22 @@ def __init__(self, model_part, settings):
 
         self.model_part = model_part
         self.model_part_name = model_part.Name
-
+        
         # the path is splitted in case it already contains a path (neeeded if files are moved to a folder)
         self.raw_path, self.raw_file_name = os.path.split(settings["input_filename"].GetString())
         self.raw_path = os.path.join(os.getcwd(), self.raw_path)
-        self.folder_name = self.raw_file_name + "__restart_files"
+
+        if settings["io_foldername"].GetString() == '':
+            self.io_foldername = self.raw_file_name + "__restart_files"
+            warn_msg  = 'No entry found for "io_foldername"\n'
+            warn_msg += 'Using the default "' + self.io_foldername + '"'
+            KratosMultiphysics.Logger.PrintWarning("SaveRestartProcess", warn_msg)
+
+        else:
+            self.io_foldername = settings["io_foldername"].GetString()
+            warn_msg  = 'Found entry found for "io_foldername"\n'
+            warn_msg += 'Using the user-defined value "' + self.io_foldername + '"'
+            KratosMultiphysics.Logger.PrintWarning("SaveRestartProcess", warn_msg)
```

Comment:
Not sure if this needs to be a warning. Aside of this, nice feature.

Reply:
Is there a more "neutral" way to remind the user? As you are right, it is not necessarily a warning. We are using similar in the drag process.